packaging/windows/sharedframework/registrykeys.wxs

```diff
@@ -13,6 +13,26 @@
           <RegistryValue Action="write" Name="1.0.0-rc2" Type="integer" Value="1" KeyPath="yes"/>
         </RegistryKey>
       </Component>
+
+<!-- Set registry keys to allow WER to genereate correct dumps-->
+      <Component Directory="TARGETDIR">
+        <RegistryKey Root="HKLM" Key="SOFTWARE\Microsoft\Windows\Windows Error Reporting\RuntimeExceptionHelperModules">
+          <RegistryValue Action="write" Name="[DOTNETHOME]shared\$(var.FrameworkName)\$(var.FrameworkDisplayVersion)\mscordaccore.dll" Type="integer" Value="0" KeyPath="yes"/>
```

Comment:
Peter let me know this is harvested, so that might be a little difficult.  Still if harvested it should have a stable ID so you could use those, but it'd be harder to read.